src/legacy/ui/public/vis/editors/default/_agg.scss

```diff
@@ -16,6 +16,9 @@
   border: $euiBorderThick;
   border-radius: $euiBorderRadius;
   background-color: transparent;
-  margin: $euiSizeS 0;
   padding: $euiSizeS;
 }
+
+.visEditorAgg__subAgg + .euiFormRow {
```

Comment:
Supply a custom class name to the EuiFormRow instead of targeting the .eui class directly.